docs/shared-content-source/docs/modules/develop/pages/cloudflow-configuration.adoc

```diff
@@ -221,7 +267,51 @@ cloudflow.runtimes.spark {
 The above example shows specific container settings for the Spark driver and executor pods. In this case the driver pods will request `1` cpu and `1Gi` of memory, limited to `2Gi` of memory and will be labeled with `key1 : value1` and 
 `key2 : value2`. The executors will be requesting `2` cpus, `2Gi` of memory, limited to `6Gi` and will be labeled with `key3 : value3`.
 
+Regards `volumes` and `volume-mounts`, `runtime.spark` is the only runtime that allows to set different `volume-mounts` to each of its pods. More concretly, while the `volumes` have to be set under `cloudflow.runtime.spark.kubernetes.pods.pod` the 'volumen-mounts' can be set specifically on the `cloudflow.runtime.spark.kubernetes.pods.driver` and/or the `cloudflow.runtime.spark.kubernetes.pods.executor`. The sample below shows how to:
+
+[source, hocon]
+----
+cloudflow.runtime.spark {
+  kubernetes.pods {
+    pod {
+     volumes {
+        foo {
+          secret {
+            name = mysecret
+          }
+        }
+        bar {
+          secret {
+            name = mysecret2
+          }
+        }
+      }
+    }
+    driver {
+      containers.container {
+        volume-mounts {
+          foo {
+            mountPath = "/mnt/file1"
+            readOnly = true
+          }
+        }
+      }
+    } 
+    executor {
+      containers.container {
+        volume-mounts {
+          bar {
+            mountPath = "/mnt/file2"
+            readOnly = true
+          }
+        }
+      }
+    } 
+  }
+}
+---- 
 
+This configuration will mount `mysecret` into any `driver` pod on path `/mnt/file1` as read only. While `mysecret2` will be mounted into any `executor` pod on path `/mnt/file2`.
```

Comment:
This is likely confusing since file1 will be the path under which more files exist, right? rather change the names of the paths.

Reply:
is confusing